Source/WebCore/platform/text/cocoa/LocaleCocoa.mm

```diff
@@ -242,10 +238,8 @@
 {
     if (!m_timeAMPMLabels.isEmpty())
         return m_timeAMPMLabels;
-    m_timeAMPMLabels.reserveCapacity(2);
     RetainPtr<NSDateFormatter> formatter = shortTimeFormatter();
-    m_timeAMPMLabels.uncheckedAppend([formatter AMSymbol]);
-    m_timeAMPMLabels.uncheckedAppend([formatter PMSymbol]);
+    m_timeAMPMLabels.appendList({ String([formatter AMSymbol]), String([formatter PMSymbol]) });
```

Comment:
Why `append` when this is known to be empty? Could we assign instead?

Reply:
Sure, will fix.